includes/wf_crm_webform_postprocess.inc

```diff
@@ -1670,6 +1670,12 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
     $params = $this->contributionParams();
     $params['contribution_status_id'] = 'Pending';
     $params['is_pay_later'] = $this->contributionIsPayLater;
+    //Fix IPN payments marked as paid by cheque
+    if (empty($params['payment_instrument_id'])) {
+      if ($params['payment_processor_id'] != 0 && $this->contribution_page['is_monetary'] == 1) {
+        $params['payment_instrument_id'] = 1;
```

Comment:
I don't think hard-coded ids are a good practice. The api explorer recommends

    'payment_instrument_id' => "Check"